docs/tutorials/google/visualizing_calibration_metrics.ipynb

```diff
@@ -155,102 +158,455 @@
     "print(\"Successful authentication to Google Cloud.\")"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "cellView": "form",
+    "id": "1Oc5MnAWSEkH"
+   },
+   "outputs": [],
+   "source": [
+    "# @title Quantum Engine Configuration\n",
+    "# @markdown Replace the `PROCESSOR_ID` and `YOUR_PROJECT_ID` with your details.\n",
+    "PROCESSOR_ID = 'weber' #@param {type: 'string'}\n",
+    "cals = cirq.google.get_engine_calibration(PROCESSOR_ID, YOUR_PROJECT_ID)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "cellView": "form",
+    "id": "QCszNh2dTW_O"
+   },
+   "outputs": [],
+   "source": [
+    "#@title View available calibration metrics\n",
+    "#@markdown See https://quantumai.google/cirq/google/calibration for a description of the available calibration metrics\n",
+    "for k, v in cals.items():\n",
+    "  print(k)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "CXo7Amk_9xUU"
+   },
+   "source": [
+    "# Metrics\n",
+    "\n",
+    "For a complete description of all the available calibration metrics, please see [Calibration metrics](../../google/calibration)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "YHOC5wMm9z6x"
+   },
+   "source": [
+    "## Single Qubit Isolated Benchmarking\n",
+    "\n",
+    "Single qubit gate error is estimated using randomized benchmarking. This error is calculated for one qubit at a time while all other qubits on the device are idle (isolated)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "dQlAa-py_D5U"
+   },
+   "source": [
+    "### Average Error\n",
+    "\n",
+    "**Disclaimer:** This data is not representative of our device's typical performance."
```

Comment:
Recommend using the same wording here as at the top: "Disclaimer: The calibration metrics shown here do not correspond to actual production metrics."

Same for subsequent disclaimers.

Reply:
done